docs/docs/building/polyfills-loader.md

```diff
@@ -0,0 +1,356 @@
+# Building >> Polyfills loader ||60
+
+> This package is still shipped via the `@open-wc` npm namespace but will soon be switch to `@web`
+
+The polyfills loader makes it easy to manage loading polyfills and/or serving different versions of your application based on browser support. It generates a script that loads the necessary polyfills and the appropriate version of the application through on runtime feature detection.
+
+A simplified version of the loader:
+
+```js
+// detect support for various browser features, load a polyfill if necessary
+var polyfills = [];
+if (!('fetch' in window)) {
+ polyfills.push(loadScript('./polyfills/fetch.js'));
+}
+
+if (!('IntersectionObserver' in window)) {
+ polyfills.push(loadScript('./polyfills/intersection-observer.js'));
+}
+
+// wait for polyfills to load
+Promise.all(polyfills).then(function() {
+ if (!('noModule' in HTMLScriptElement.prototype)) {
+ // browser doesn't support es modules, load a SystemJS build with es5
+ System.import('./legacy/app.js');
+ } else {
+ // browser supports modules, import a modern build
+ import('./app.js);
+ }
+});
+```
```

Comment:
this code snippet is not formatted

Reply:
I copied the original source and as it has a missing `'` it did not get auto formatted 😅

good cached 🤗  fixed now 👍

and always nice that prettier only works if the code is valid 😬